yarn.config.cjs

```diff
@@ -1,3 +1,4 @@
+// @ts-check
```

Comment:
Happy if want to remove this. I just use it as a guard (I can't go back to JS 🙈)